### PR TITLE
Feat/user profile

### PR DESCRIPTION
**Pull request by hansac91**

### CODE REVIEW

File: app/(profile)/user-profile/page.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styles from './page.module.scss'
+import { getServerSession } from 'next-auth'
+import { authOptions } from '@/lib/auth'
+import { redirect } from 'next/navigation'
```
Comment (wawrzonnn) [inline app/(profile)/user-profile/page.tsx:5]: session, auth, redirect - - we don't need them here.

**Review by wawrzonnn — APPROVED** [review]
Approved, but please clean up the page.tsx

File: app/(profile)/user-profile/page.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import UserProfileMain from '@/components/UserProfile/UserProfileMain/UserProfileMain'
+import UserProfileDetails from '@//components/UserProfile/UserProfileDetails/UserProfileDetails'
+
+const UserProfilePage = async () => {
+  const session = await getServerSession(authOptions)
+
+  if (!session) {
```
Comment (Sumick) [inline app/(profile)/user-profile/page.tsx:12]: **issue:** I think we want to preview user profiles now as unauthorized users

**Review by Sumick — APPROVED** [review]
Looks good, but let's remove redirection for unauthorized users for previewing user profie

**Review by AniaPiWo — APPROVED** [review]

**Pull request merged**